docs/operating-scylla/admin.rst

```diff
@@ -80,8 +80,8 @@ The following addresses can be configured in scylla.yaml:
      - Address for REST API requests. See api_port in the :ref:`Networking <cqlsh-networking>` parameters.
    * - prometheus_address
      - Address for Prometheus queries. See prometheus_port in the :ref:`Networking <cqlsh-networking>` parameters and `ScyllaDB Monitoring Stack <https://monitoring.docs.scylladb.com/stable/>`_ for more details.
-   * - replace_address_first_boot
-     - Address of the node this Scylla instance is meant to replace. Refer to :doc:`Replace a Dead Node in a Scylla Cluster </operating-scylla/procedures/cluster-management/replace-dead-node>` for more details.
+   * - replace_node_first_boot
+     - Host ID of the node this Scylla instance is meant to replace. Refer to :doc:`Replace a Dead Node in a Scylla Cluster </operating-scylla/procedures/cluster-management/replace-dead-node>` for more details.
```

Comment:
Just in this sentence we use node, instance and host for the same thing. Could we settle on fewer names for the same thing perhaps or provide a glossary of what the differences are? At the very least we perhaps should consistently use "Scylla node" rather than "instance" in all places.

Reply:
Makes sense.  How about:

```
  * - replace_node_first_boot
     - Host ID of a dead node this Scylla node is replacing. Refer to ...
```

Reply:
Rephrased in [6808aa1](https://github.com/scylladb/scylladb/commit/6808aa1bda5f9f464683dd1e4e5a158d17221344)

Reply:
It still says "Host ID of the node this Scylla instance is meant to replace"